staging/src/k8s.io/client-go/tools/remotecommand/remotecommand_test.go

```diff
@@ -264,3 +300,65 @@ func v4WriteStatusFunc(stream io.Writer) func(status *apierrors.StatusError) err
 		return err
 	}
 }
+
+func newStreamExecutor(config *rest.Config, method string, url *url.URL) (*streamExecutor, error) {
+	wrapper, upgradeRoundTripper, err := spdytransport.RoundTripperFor(config)
+	if err != nil {
+		return nil, err
+	}
+	return &streamExecutor{
+		upgrader:  upgradeRoundTripper,
+		transport: wrapper,
+		method:    method,
+		url:       url,
+		protocols: []string{
+			remotecommandconsts.StreamProtocolV4Name,
+			remotecommandconsts.StreamProtocolV3Name,
+			remotecommandconsts.StreamProtocolV2Name,
+			remotecommandconsts.StreamProtocolV1Name,
+		},
+	}, nil
+}
+
+func TestStreamExitsAfterConnectionIsClosed(t *testing.T) {
```

Comment:
Can you add a comment on why we need this test? Something like "StreamWithContext closes the conn when the context is done. This test verifies that the close of the conn leads to the termination of the stream.stream() goroutine running in the background".

Reply:
Thought I think the test name is meaningful enough, it's fine to add such comments.

Reply:
Thanks.